spec/modules_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe 'Wprof Generate Profiling' do
+  context 'When include Wprof to a class' do
+    before do
+      allow(Rails).to receive_messages(wprof: { custom_methods: ['foo_method'] })
+      require 'support/mocks_and_stubs/class_for_modules'
+      allow(GenProf).to receive_messages(deploy_reporter: self)
+      stub_request(:get, 'http://www.testexternal.com/external')
+    end
+    let(:test_class) { ClassForModules.new }
+
+    it 'Wprof must be wrapper httparty methods' do
+      expect(test_class.consume_external_service[:subscripted_to]).to eq('wprof.service')
+    end
+    it 'Wprof must be wrapper custom method' do
+      params = test_class.foo_method
+      expect(params[:subscripted_to]).to eq('wprof.custom')
```

Comment:
It's not usually a good practice to have multiple expects in one spec either (use a before and just add the expect in each spec, stuff like that)

Reply:
It's true, but isn't black or white. If you are testing a state for an object when that state is only true if has some conditions, so it's valid to use more than one assertion for test it. For improve each assert in the case of these failed, I put comments for each one. That must be enough.